samples/interceptors/.gitignore

```diff
@@ -1 +1,2 @@
 bookstore/
+certs/
```

Comment:
Maybe we have to commit certs to the repo, to try out the scenario on the fly without generating certs. Will add with later changes, +1 for now